### PR TITLE
feat: different types of buttons

### PR DESCRIPTION
**Pull request by amarphule**
Created Different types of buttons
1. Buttons with a solid background
2. Buttons with the outline
3. Buttons with icon
4. Added hover effect accordingly
buttons

### CODE REVIEW

**Review by ishapatell1 — CHANGES_REQUESTED** [review]
The text inside button seems small. Please do the necessary changes.

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  background-color: transparent;
+  font-size: 1.2rem;
+  font-weight: 700;
+  border-radius: 5px;
+  padding: 0.5rem 1rem;
+  border: none;
+  cursor: pointer;
+  margin-bottom: 1rem;
+}
+/*
```
Comment (prathmeshjagtap) [inline components/button.css]: Remove comments  while merging. Rest code looks clean and good

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

**Comment by amarphule** [comment]
> The text inside button seems small. Please do the necessary changes.

@Isha2806  It's 1.2rem, I think it's fine for button text size. If I increase the font size of button text it doesn't look good. And my Heading has font size 3rem, so I kept button text size considering heading and description font size.

File: components/button.css (hunk 1 above)
Comment (amarphule) [reply components/button.css]: Thanks for pointing it out. Removed unnecessary comments

**Pull request merged**